src/Psr/Uri.php

```diff
@@ -10,16 +10,16 @@
 use Psr\Http\Message\UriInterface;
 
 /**
- * Class URI
+ * Class Uri
  *
  * This is an adapter class that implements the PSR-7 UriInterface that can't be implemented
- * by the URL class itself because it isn't designed to be immutable.
+ * by the `Url` class itself because it isn't designed to be immutable.
  */
 
 class Uri implements UriInterface
 {
     /**
-     * @var URL
+     * @var Uri
```

Comment:
It's an instance of the `Url` class, not the `Uri` class.

Reply:
```suggestion
     * @var Url
```